internal/common/common.go

```diff
@@ -12,6 +12,10 @@ func HdrValueAuthorization(key string) string {
 	return "Bearer " + key
 }
 
+func BasePath(server string) string {
```

Comment:
This is not a path 🤔 
This is a full URI what we return here.
Given that the `server` string is a valid URL 🙃 

We can improve this later by checking this for an valid URL 👍 

However, for now I just want to rename it.
Something like `ServerApiUrl`? 🤔 Or something... Path feels wrong here...

Reply:
Well first of all too much checking and validations obscure the business logic. IMO it should be the responsibility of the user to make sure `httttttp://he/is/not/passing/invalid/url` :smile: . This aligns with our goal of keeping our API and CLI as  just thin wrappers.

But yes the function name is not apt but then IMO the new name `BaseURL` is more apt because we will be `extending` it.